examples/setup-aep-fee-router/index.ts

```diff
@@ -0,0 +1,283 @@
+import {
+  createPublicClient,
+  createWalletClient,
+  http,
+  defineChain,
+  parseAbi,
+  getAddress,
+} from 'viem';
+import { privateKeyToAccount } from 'viem/accounts';
+import { arbitrumSepolia } from 'viem/chains';
+import {
+  feeRouterDeployChildToParentRouter,
+  feeRouterDeployRewardDistributor,
+  createRollupFetchCoreContracts,
+  createTokenBridgeFetchTokenBridgeContracts,
+  arbOwnerPublicActions,
+  arbGasInfoPublicActions,
+} from '@arbitrum/orbit-sdk';
+import { sanitizePrivateKey, getParentChainLayer } from '@arbitrum/orbit-sdk/utils';
+import { config } from 'dotenv';
+config();
+
+// environent variables check
+if (typeof process.env.ROLLUP_ADDRESS === 'undefined') {
+  throw new Error(`Please provide the "ROLLUP_ADDRESS" environment variable`);
+}
+
+if (typeof process.env.CHAIN_OWNER_PRIVATE_KEY === 'undefined') {
+  throw new Error(`Please provide the "CHAIN_OWNER_PRIVATE_KEY" environment variable`);
+}
+
+if (typeof process.env.ORBIT_CHAIN_ID === 'undefined') {
+  throw new Error(`Please provide the "ORBIT_CHAIN_ID" environment variable`);
+}
+
+if (typeof process.env.ORBIT_CHAIN_RPC === 'undefined') {
+  throw new Error(`Please provide the "ORBIT_CHAIN_RPC" environment variable`);
+}
+
+// load the chain owner account (or any account that has the executor role in the UpgradeExecutor)
+const chainOwner = privateKeyToAccount(sanitizePrivateKey(process.env.CHAIN_OWNER_PRIVATE_KEY));
+
+// set the parent chain and create a public client for it
+const parentChain = arbitrumSepolia;
```

Comment:
shouldn't this be provided as an input?

Reply:
below the script appears to assume the parent chain is mainnet (hardcoded 0x1Afa41C006dA1605846271E7bdae942F2787f941 musig address)

Reply:
I've added the parent chain target address as an env variable. All these scripts are meant to be only examples that one can copy and adjust as necessary, and they follow the same structure: parent chain arbSepolia and inputs as env variables or random information (like the chain id, batch poster and staker wallet when creating a rollup for example). Happy to discuss further though

Reply:
Examples were modified so all inputs are now defined in the .env file.